database/mysql/createdatabase_generic.sh

```
@@ -62,14 +65,23 @@ echo "loading icd${ICD}_issue_groups.sql..."
+bhash=$(htpasswd -bnB carlosdoc "${newpassword}" | cut -d: -f2)
+bhash="{bcrypt}${bhash}"
+echo "UPDATE security SET password='${bhash}' WHERE user_name='carlosdoc';" | $mysql_cmd
+newpin=$(tr -cd '0-9' < /dev/urandom | fold -w4 | head -n 1)
+echo "UPDATE security SET pin='${newpin}' WHERE user_name='carlosdoc';" | $mysql_cmd
+
 echo 'all done!'
 echo 'the default user is carlosdoc'
-echo 'password carlos2026'
-echo 'pin 1117'
-echo 'For sec reasons these credentials are set to expire in a month!'
+echo 'password '${newpassword}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)